### PR TITLE
Fix the order of error term's operands

### PR DESCRIPTION
**Pull request by wlbksy**
Error term should be "actual minus predicted".

This PR should be no op, since all related error terms are either squared or transformed into absolute value.

### CODE REVIEW

File: python/mxnet/gluon/loss.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def hybrid_forward(self, F, x, *args, **kwargs):
 class L2Loss(Loss):
     r"""Calculates the mean squared error between `pred` and `label`.
```
Comment (vishaalkapoor) [inline python/mxnet/gluon/loss.py]: by the same reasoning, this should be swapped as `label` is the subject.

Comment (vishaalkapoor) [reply python/mxnet/gluon/loss.py]: same elsewhere.

Comment (wlbksy) [reply python/mxnet/gluon/loss.py]: Thanks, good finding!

Comment (vishaalkapoor) [reply python/mxnet/gluon/loss.py]: Thanks!

**Comment by vishaalkapoor** [comment]
LGTM!

**Pull request merged**